setup.py

```diff
@@ -39,7 +39,7 @@ def get_version(file, name='__version__'):
     zip_safe=False,
     packages=find_packages(exclude=('tests', 'docs')),
     install_requires=[
-        'topopy>=0.1', 'numpy', 'sklearn', 'pandas'
+        'nglpy>=1.0.6,<2.0', 'topopy>=1.0,<2.0', 'numpy', 'sklearn', 'pandas'
```

Comment:
It is up to you, but it may be easier to load all of the `requirements.txt` and fill this section in here, so you don't have to constantly update twice. They do have different meanings, I suppose, but for all intents and purposes I think a `python setup.py` or a `pip` should grab the prerequisites for you. 🤷‍♂